cpp/src/text/subword/subword_tokenize.cu

```diff
@@ -125,6 +129,33 @@ __global__ void kernel_compute_tensor_metadata(
   }
 }
 
+struct tabulator {
+  uint32_t __device__ operator()(cudf::size_type idx) { return ((idx % 3) == 0) ? idx : 0; }
```

Comment:
```suggestion
  uint32_t __device__ operator()(cudf::size_type idx) const { return ((idx % 3) == 0) ? idx : 0; }
```

Reply:
I even don't see a reason to have this functor. It is small, and used just once. So technically we can just use a lambda directly.